.gitignore

```diff
@@ -14,17 +16,13 @@ ruby_sess.*
 *.pyc
 .svn
 .class
-# except foo.o which is hand maintained
-!foo.o
-.DS_Store
 .idea/VexFlow.iml
 .idea/encodings.xml
 .idea/misc.xml
 .idea/modules.xml
 .idea/scopes/scope_settings.xml
 .idea/vcs.xml
 .idea/workspace.xml
-node_modules
```

Comment:
Why did you remove `node_modules`?

Reply:
I moved it up in the same file. Sorry for the confusion.